frontend/src/translations/result/en.json

```diff
@@ -33,6 +33,8 @@
   "payment.creditcard": "Credit Card",
   "payment.paypal.title": "Make a new payment with Paypal",
   "payment.paypal.description": "Remember that the person assigned for this task will receive the payment with Paypal as well.",
+  "payment.paypal.warning": "Pagamentos realizados pelo Paypal estão sujeitos a cobranças da plataforma.",
```

Comment:
We don't need to add translations strings in this file, it's a generated file by other tool